docs/writing_reporters.md

```diff
@@ -106,6 +125,13 @@ def handle_event(_event_name, measurements, metadata, metrics) do
 end
 ```
 
+The implementation of `is_filtered?/2` might look like:
+
+```elixir
+defp is_filtered?(%{filter: nil}, _metadata), do: false
+defp is_filtered?(metric, metadata), do: metric.filter.(metadata)
```

Comment:
In Elixir it is either `is_filtered` Or `filtered?` without is. Giving this is not a guard, I recommend the latter.